api/HTMLHeadingElement.json

```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
+    "HTMLHeadingElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLHeadingElement",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": true
+          },
+          "firefox_android": {
+            "version_added": true
```

Comment:
[Should be `4`.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHeadingElement#Browser_compatibility)